README.md

```diff
@@ -29,6 +29,26 @@ Because the WordPress.org plugin repository shows information from the readme in
 * `ASSETS_DIR` - defaults to `.wordpress-org`, customizable for other locations of WordPress.org plugin repository-specific assets that belong in the top-level `assets` directory (the one on the same level as `trunk`).
 * `README_NAME` - defaults to `readme.txt`, customizable in case you use `README.md` instead, which is now quietly supported in the WordPress.org plugin repository.
 
+## Example Git Workflow
+
+For this example, Git's `main` branch (the default on new repositories) corresponds directly to Subversion's `trunk` and is considered the "release" branch.
+
+In general, the expected workflow is as follows:
+
+1. Create a new branch for feature x
+2. When the feature is ready for release, merge it onto `main`
+3. On `main`, update the README to reflect the change, e.g.: set the "Stable tag", update the changelog, etc. (although some of this could be done on the feature branch prior to merging)
```

Comment:
```suggestion
3. On `main`, update the `readme.txt` to reflect the change, e.g.: set the "Stable tag", update the changelog, etc. (although some of this could be done on the feature branch prior to merging)
```